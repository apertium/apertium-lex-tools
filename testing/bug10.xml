<rules>

  <rule>
    <match lemma="kazati" tags="vblex.*"><select lemma="рече"/></match>
  </rule>

  <rule weight="3">
    <match                tags="*"/>
    <match lemma="kazati" tags="vblex.*"><select lemma="каже"/></match>
    <match lemma="."      tags="sent"/>
  </rule>

</rules>
