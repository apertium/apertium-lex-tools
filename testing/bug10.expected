^kazati<vblex><perf><iv><lp><m><sg>/рече<vblex><perf><tv><lp><m><sg>$ ^biti<vbser><clt><pres><p3><sg>/е<vbser><pres><p3><sg>$ ^je<d>/je<s>$ ^kazati<vblex><perf><iv><lp><m><sg>/каже<vblex><perf><tv><lp><m><sg>$^.<sent>/.<sent>$
